docs/basic-features/built-in-css-support.md

```diff
@@ -1,17 +1,127 @@
 ---
-description: Next.js includes styled-jsx by default for isolated and scoped CSS support, but you can also use any other CSS-in-JS solution!. Learn more here.
+description: Next.js supports including CSS files as Global CSS or CSS Modules, using `styled-jsx` for CSS-in-JS, or any other CSS-in-JS solution! Learn more here.
 ---
 
-# Built-in CSS Support
+# Built-In CSS Support
+
+Next.js allows you to import CSS files from a JavaScript file.
+This is possible because Next.js extends the concept of [`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) beyond JavaScript.
+
+## Adding a Global Stylesheet
+
+To add a stylesheet to your application, import the CSS file within `pages/_app.js`.
+
+For example, consider the following stylesheet named `styles.css`:
+
+```css
+body {
+  font-family: 'SF Pro Text', 'SF Pro Icons', system-ui;
+  padding: 20px 20px 60px;
+  max-width: 680px;
+  margin: 0 auto;
+}
+```
+
+Create a [`pages/_app.js` file](https://nextjs.org/docs/advanced-features/custom-app) if not already present.
+Then, [`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) the `styles.css` file.
+
+```jsx
+import '../styles.css'
+
+// This default export is required in a new `pages/_app.js` file.
```

Comment:
This line is not required, we already warn about always adding a default export in the introduction for pages and the section for `_app`.
```suggestion
```

Reply:
I think this comment should remain so users don't prune this code and think it's optional. 

When adding stylesheets, they may not have read any other docs yet.

Reply:
I agree with you, someone may want to just keep the import and remove the component. 👍 

On another note, we trust that our users know what are pages at the least. Pages are a prerequisite or otherwise they won't have the context for this entire section (and many others), but because pages are explained a lot at the start of our docs we trust they know it (or otherwise a lot of our docs are wrong).

Reply:
Custom apps are only explained in the advanced docs tho, so IMO we should stress that _this specific_ default export is required.

Otherwise, they may put a random component.